tests/plotting/jupyter/test_trame.py

```diff
@@ -53,7 +53,11 @@ def test_trame():
     actor = pl.add_mesh(pv.Cone())
     widget = pl.show(return_viewer=True)
     assert isinstance(widget, Widget)
-    assert 'http://' in widget.src
+
+    if pv.global_theme.trame.server_proxy_enabled:
+        assert pv.global_theme.trame.server_proxy_prefix in widget.src
+    else:
+        assert 'http://' in widget.src
```

Comment:
Intentionally snuck in this PR, or misplaced commit?

Reply:
Intentionally snuck in as the changes here fix other failures in the same testing module